src/fr/japanread/src/eu/kanade/tachiyomi/extension/fr/japanread/bentomanga.kt

```diff
@@ -24,11 +24,13 @@ import rx.Observable
 import uy.kohesive.injekt.injectLazy
 import java.util.Calendar
 
-class Japanread : ParsedHttpSource() {
+class bentomanga : ParsedHttpSource() {
```

Comment:
Class names should start with uppercase.

```suggestion
class BentoManga : ParsedHttpSource() {
```

Rename the file to `BentoManga.kt` as well.